### DISCUSSION TITLE
Cannot create buckets in Supabase

### DISCUSSION THREAD

**Discussion by sumchans** [supabase/supabase#2596] [Questions] [answered]
Not sure if anybody is having this issue, or if I am having a setting off on my side. Since yesterday I have been trying to create a bucket. I keep getting this message when I click Create Bucket **An invalid response was received from the upstream server**

**Comment by inian** [comment, marked as answer]
Can you send the project ref to beta@supabase.io @sumchans? I will take a look

**Reply by sumchans** [reply]
@inian where can I find the reference you are talking about in my project.

**Reply by inian** [reply]
saw your support ticket just now Sumesh. That is enough information for us to take a look. Lets continue the conversation there

**Reply by sumchans** [reply]
Okay thanks let me know.

**Comment by sumchans** [comment]
I have sent it, please check it out and let me know if that’s all you need .

**Comment by itsrishabh** [comment]
Also having the same issue here – cannot see, create, or remove buckets
![image](https://github.com/supabase/supabase/assets/2372098/d9792b8f-b672-4fb7-8eec-886d2357d00f)
![Screenshot 2023-07-17 at 6 07 58 PM](https://github.com/supabase/supabase/assets/2372098/29c5dc5d-5918-43ef-843f-757a07b654a6)
